src/common/transform_json.py

```diff
@@ -20,6 +22,71 @@ def search_evs_list(data, search_str):
         if search_str in item.keys():
             return item[search_str]
 
+def find_ncit_code(data, ncit_key):
+    values = []
+    
+    def search_dict(d):
+        if isinstance(d, dict):
+            for key, value in d.items():
+                if key == ncit_key:
+                    values.append(value)
+                elif isinstance(value, dict):
+                    search_dict(value)
+                elif isinstance(value, list):
+                    for item in value:
+                        search_dict(item)
+    
+    search_dict(data)
+    return values
+
+def extract_transform_model_cde_data(props):
+    mongo_list = []
+    for prop in props["PropDefinitions"].keys():
+        try:
+            terms = props["PropDefinitions"][prop].get("Term")
+            prop_defination = props["PropDefinitions"][prop]["Desc"]
+            mongo_list.append(f'{prop} is {prop_defination}')
+            if terms is None:
+                continue
+            else:
+                cde_code = terms[0].get("Code")
+                if cde_code is None:
+                    continue
+                cde_url = "https://cadsrapi.cancer.gov/rad/NCIAPI/1.0/api/DataElement/" + cde_code
```

Comment:
Could you create a generic utility function for API call, so later it can be reused.